Gutenberg/cocoapods_helpers.rb

```diff
@@ -56,7 +56,8 @@
   RNCClipboard
   RNFastImage
   React-Codegen
-  React-bridging
+  React-jsc
+  React-hermes
```

Comment:
On iOS we don't support Hermes yet, so I'm wondering if need to add this pod 🤔 .